README.md

```diff
@@ -670,16 +654,38 @@ async function startCheckout() {
                 console.error(error.name, error.message, error.stack, component);
                 handleResponse(error, component);
             },
-            // Step 13 onAdditionalDetails(...) - Used for Native 3DS
-            onAdditionalDetails: async (state, component, actions)=> {
-                const response = await sendPostRequest("/api/payments/details", state.data);
-                handleResponse(response, component);
+            // Uncomment this if you want to add Native 3DS
```

Comment:
FYI: I need to make some changes to this function -> We need to accept/reject as follows: https://github.com/adyen-examples/adyen-java-spring-online-payments/blob/main/checkout-example-advanced/src/main/resources/static/dropin.js#L66-L87

We no longer need to check the response.action part